article/app/services/dotcomponents/ArticlePicker.scala

```diff
@@ -96,12 +97,27 @@ object ArticlePicker {
       ("dcrCouldRender" -> dcrCanRender.toString) +
       ("pageTones" -> pageTones)
 
-    if (tier == RemoteRender) {
+    if (tier == DotcomRendering) {
       DotcomponentsLogger.logger.logRequest(s"path executing in dotcomponents", features, page)
+    } else if (tier == PressedArticle) {
+      DotcomponentsLogger.logger.logRequest(s"Article served from pressed content", features, page)
```

Comment:
I'm not entirely sure here but it might be safer to add metadata to record this information rather than change this balance?

It's hard to say. I can see how what you've done makes a lot of sense in terms of the fact there are now 3 places the code can go to render the article. On the other hand, this text is used to produce a number of graphs that provide information around DCR/Frontend traffic patterns and this change would impact them.

On yet another hand, are we using those graphs anymore?

In any case, if you did decide to go with this approach - and I'm happy to leave this choice up to you - is there value in keeping the text consistent? Eg. less `Article served` more `path executing`.

Reply:
We will have to update the dashboards to reflect the percentage of pressed content (as a follow on task). After the migration to 100% Articles, we are expecting to have 0 content rendered from frontend. We expect the frontend tier to act as a fallback and all the content to be rendered by either DCR or pressed. 

This feels like an intermediate step until the frontend code gets deleted. After that we will only have 2 rendering tiers and at that point we will have to refactor both the logging and the dashboards again.